Data/Set.hs

```diff
@@ -13,14 +13,48 @@
 -- Maintainer  :  libraries@haskell.org
 -- Portability :  portable
 --
--- An efficient implementation of sets.
+--
+-- = Sets
+--
+-- The @'Set' e@ type represents a set of /unique/, /ordered/ elements of type
+-- @e@.
```

Comment:
What are ordered elements? What is a unique element? I think perhaps

    -- The @'Set' e@ type represents a set of elements of type @e@. Most operations
    -- require that @e@ be an instance of the 'Ord' class.

Reply:
Done.